frontend/src/store/auctions.ts

```diff
@@ -0,0 +1,33 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import { AuctionItem } from "interfaces";
+import { fetchAuctions } from "./helpers/auctionsHelper";
+
+const initialState = {
+    status: "",
+    auctions: [] as AuctionItem[],
+};
+
+export const getAuctions = createAsyncThunk("auctions/setAuctions", () => {
+    return fetchAuctions().then((data) => data.data);
```

Comment:
suggestions: change `data` to something more descriptive like `auctionsResponse`, to avoid this weird `data.data` chain